emails/views.py

```diff
@@ -352,11 +352,11 @@ def _sns_message(message_json):
         incr_if_enabled('email_with_html_content', 1)
         wrapped_html = render_to_string('emails/wrapped_email.html', {
             'original_html': html_content,
+            'recipient_profile': user_profile,
```

Comment:
```suggestion
            'recipient_language': user_profile.language,
```

Reply:
I like putting "fat" objects into templates rather than fine-grained values. There's minimal performance impact (unless the object is VERY large), and it makes future code easier. So when future Luke opens this template again, he will see that `recipient_profile` is available, and can use any other property on it.